crd.txt

```diff
@@ -0,0 +1,6 @@
+GlobalRole.management.cattle.io
```

Comment:
nit: Rename to resources.sample.txt to be explicit that this is an example. The docs team will need to adapt this file based on the Rancher's version (since not all Rancher's version will have the same list of resources to look at.

(Note that I changed the prefix from crd to resources because those aren't all going to be CRDs => Tokens.ext.cattle.io is NOT a CRD)

Reply:
Added polling logic. PTAL.